src/searchbar.cpp

```diff
@@ -79,7 +79,9 @@ SearchBar::SearchBar(QWidget *parent) :
 
 void SearchBar::on_currentTitleChanged(const QString& title)
 {
-    setText(title);
+    if (!title.startsWith("zim://")) {
+        setText(title);
+    }
```

Comment:
We should display an empty string (or something else) if title doesn't start with "zim://". This is what we do in the search bar (https://github.com/kiwix/kiwix-desktop/blob/master/src/tabbar.cpp#L147-L153)

The `url.path()` is better than nothing, but if the `url` is empty (or `/`) it would be even better to display the zim name. (But it should be in another issue/pr).

Reply:
@mgautierfr I'm not so sure how @jetownfeve21 applies in real-life... If we keep it like it is, are we sure something will go wrong?

Reply:
The problem is that if we do nothing, we will keep the title of the previous page.

Reply:
So for now I display an empty string and maybe in another pr it will be the zim name or something else ?

Reply:
yes